install_config/cluster_metrics.adoc

```diff
@@ -289,6 +289,12 @@ ifdef::openshift-origin[]
 /usr/share/openshift/examples/infrastructure-templates/origin/metrics-deployer.yaml
 ----
 ====
+
+[NOTE]
+Depending on your installation method, the template may not be present in your OpenShift installation. If so, the template can be found at the following GitHub location:
```

Comment:
I know we're not doing this anymore...

What's the reasoning behind having to wrap again?